src/posts/announce-2.2.1.Final.adoc

```diff
@@ -0,0 +1,117 @@
+---
+title: Announcing Thorntail 2.2.1.Final
+publishDate: 2018-10-30
+modifyDate: 2018-10-30
+author: The Thorntail Team
+layout: post.jade
+---
+
+== Thorntail 2.2.1.Final
+
+Winter is coming! (at least in the Northern Hemisphere) but we can stay warm with another Thorntail release! Say "hello" to Thorntail 2.2.1.Final!
+
+=== What's New?
+
+This release contains several bugfixes from different areas of the product, such as MicroProfile Config, MP RestClient or Topology used in OpenShift.
+MicroProfile OpenAPI now works using Swarm.run() from IDE.
+There were other improvements around dependencies in some fractions.
+
+++++
+<!-- more -->
+++++
+
+== Deprecation and config keys breaking changes
```

Comment:
I would split this into two. Deprecating `@CreateSwarm` has nothing in common with the `swarm.*` to `thorntail.*` change.